cli/tests/e2e/snapshots/test_ci/test_dryrun/results.txt

```diff
@@ -192,6 +198,12 @@ Would have sent findings and ignores blob: {
             },
             "is_blocking": false,
             "match_based_id": "2c4ff12fcdf80ef1c00dd0f566ae102d792c7ba68e560d70f111aae3b3216c0b1b943e74d2ce29c0361f1fbc37bd4e9aafd32c3435a36c61b8bd3963efe0d7a1_0",
+            "hashes": {
```

Comment:
This is an SCA finding - is it expected that these are all null?

Reply:
Sort of - it's an artifact of the test set-up being kind of incomplete. I could flesh out all of the tests to have real rules with real finding files, but that's a bit more of an undertaking. Since this test isn't testing specifically for these hashes, I downscoped that out, but happy to revisit if you disagree.